pkgs/applications/editors/neovim/default.nix

```diff
@@ -18,16 +18,8 @@
 let
   requiredLuaPkgs = ps: (with ps; [
     lpeg
-    luabitop
-    mpack
   ] ++ lib.optionals doCheck [
```

Comment:
this is a dead branch since it's not used in nixpkgs but I keep it as maintainer to help debug stuff. We do use busted at least in the code so I dont think we should remove it ? or is the system busted completely ignored even when available ? kinda makes me sad but as it concerns tests, it's less of an issue

Reply:
I don't think it's possible to run tests with external busted at the moment. Still, that will probably be fixed later on so I it should probably stay. The others I don't think are needed at all.

Reply:
are these meaningless then https://github.com/neovim/neovim/blob/09a17f91d0d362c6e58bfdbe3ccdeacffb0b44b9/cmake.deps/CMakeLists.txt#L35 ? distributions are keen on using system libraries since it makes security updates, code review easier. I die a little bit inside everytime neovim vendors something (especially if it's just to avoid luarocks :s ). debian has a hardcore stance against vendoring, nixpkgs is more flexible but let's use system libraries if possible

Reply:
Well, no, because those are build dependencies rather than test dependencies. Using external build deps works fine.

Reply:
> nixpkgs is more flexible but let's use system libraries if possible

I think we already are? The things I removed are either outdated or only for tests.